plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/internal/UINavigatorMessages.java

```diff
@@ -229,6 +229,7 @@
     public static String question_no_sql_available;
     public static String error_sql_generation_title;
     public static String error_sql_generation_message;
+    public static String dialog_search_objects_search_in_comments;
```

Comment:
Consider using `dialog_project_goto_object_` prefix for `GotoObjectDialog`-related localization strings and move it here:
https://github.com/dbeaver/dbeaver/blob/0f8c378219dcf94f6dc5643772219f9dcf2cf60e/plugins/org.jkiss.dbeaver.ui.navigator/src/org/jkiss/dbeaver/ui/internal/UINavigatorMessages.java#L80-L81